### DISCUSSION TITLE
How do we propose leveraging CALM to represent planned changes in architecture?

### DISCUSSION THREAD

**Discussion by LeighFinegold** [finos/architecture-as-code#465] [Q&A] [answered]
It is often required to communicate a vision of where you want your architecture to be to a general audience. How would this be represented in CALM? Would we recommend separate core document per phase (still sharing controls/patterns where required)?

**Comment by Thels** [comment]
I like the idea of having a core document per phase - potentially with a "evolution" schema sitting on top of it so that the evolution over time  can be captured, without resorting to source-control.

**Reply by LeighFinegold** [reply, marked as answer]
In the end https://github.com/finos/architecture-as-code/issues/1409 was raised with an implementation on https://github.com/finos/architecture-as-code/pull/2104. The term phases was replaced with the concept of a moment. With the timeline document specifying the current moment. 

There is current thinking to support multiple potential timelines on https://github.com/finos/architecture-as-code/issues/2733

**Comment by jpgough-ms** [comment]
@LeighFinegold this is a great question. From my perspective, this sits somewhere in-between what C4 provides and CALM. 

I think there is room to have nodes in CALM with a detailed-architecture that is not specified at the time of design, this can then be filled out later.
